cg/cli/workflow/balsamic/base.py

```diff
@@ -199,7 +199,7 @@ def start(
     analysis_api: BalsamicAnalysisAPI = context.obj.meta_apis["analysis_api"]
     analysis_api.prepare_fastq_files(case_id=case_id, dry_run=dry_run)
     LOG.info(f"Starting analysis for {case_id}")
-    context.invoke(link, case_id=case_id, dry_run=dry_run)
+    context.invoke(link, case_id=case_id, dry_run=False)
```

Comment:
We need the folder the link generates to create de `config.json` file. This is not adding a new functionality, we just want to have the previous behaviour back (we introduced a bug when removing the unnecessary `-r` flag)

Reply:
but then this is not an actual dry-run, as you are creating files. You could do config-case, generate the configs and then dry-run on the `run` function to get this.

Reply:
Maybe the previous behaviour was not ideal

Reply:
But it's fine, I can sacrifice myself and run two commands instead of one if I want the config 🤣 😜 

Will ask for a re-review after changing the behaviour. Thank you both!

Reply:
aha, then wouldn't it make sense to add the `context.invoke(link, case_id=case_id, dry_run=dry_run)` within config-case? it seems you need to do that to fully config the case

Reply:
that wouldn't solve the issue, but it would be 2 commands only to run, instead of 3